barretenberg/cpp/src/barretenberg/ultra_honk/merge_verifier.cpp

```diff
@@ -5,7 +5,7 @@ namespace bb {
 template <typename Flavor>
 MergeVerifier_<Flavor>::MergeVerifier_()
```

Comment:
It makes no sense for these Marge classes to be templated by Flavor. I removed this a few weeks ago. Just now noticing that someone added it back. Annoying